app/process-tasks.js

```diff
@@ -60,6 +61,8 @@ function executeWorkerForTaskType (worker, task) {
               })
             }
           })
+        } else if (task.type === taskType.CALCULATE_WORKLOAD_POINTS && task.additionalData.operationType === wpcOperationType.UPDATE) {
+          return callWebRefreshEndpoint()
```

Comment:
Minor: You could wrap the the operationType check inside an if statement for the the task.type, rather than checking the task.type twice.